docs/pages/docs/context/get-context.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Get Context"
+description: "getContext is a function exported by @keystone-6/core/context to support operations without starting a HTTP server"
+---
+
+Keystone's command line interface is helpful, but sometimes you don't want to startup the HTTP server, or kickoff any build processes.
+Maybe you need to insert data into the database, maybe you only want Keystone for your internal schema management behind a custom protocol or tiny REST API.
+*Or maybe you want to write unit tests.*
+
+If you have previously built Keystone using `keystone build` or used `keystone dev`, then you can use `getContext`.
```

Comment:
Should we clarify what _built_ means here? Something along the lines of _if you have already generated the schema and prisma client (and whatever build command does) using `keystone build` or `keystone dev`_?

Reply:
I don't think so, not unless we want to introduce documentation the explains each of the artifacts of Keystone - the hint is that you need to run `keystone build` or `keystone dev`, I think we can leave at that.

I might change the wording to remove `built` completely